lib/fastlane/plugin/appcenter/actions/appcenter_fetch_version_number.rb

```diff
@@ -0,0 +1,99 @@
+require 'json'
+require 'net/http'
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Actions
+    class AppcenterFetchVersionNumberAction < Action
+      def self.description
+        "Fetches the latest version number of an app from App Center"
+      end
+
+      def self.authors
+        ["jspargo", "ShopKeep"]
+      end
+
+      def self.run(params)
+        api_token = params[:api_token]
+        app_name = params[:app_name]
+        owner_name = params[:owner_name]
+
+        if owner_name.nil?
+          owner_name = get_owner_name(api_token, app_name)
+        end
+
+        releases = Helper::AppcenterHelper.fetch_releases(
+          api_token: api_token,
+          owner_name: owner_name,
+          app_name: app_name
+        )
+
+        UI.abort_with_message!("No versions found for '#{app_name}' owned by #{owner_name}") unless releases
+        sorted_release = releases.sort_by { |release| release['id'] }.reverse!
+        latest_build = sorted_release.first
```

Comment:
Minor: Builds and releases are different things in App Center. You're dealing only with releases here.